bgpd/bgp_table.c

```diff
@@ -83,16 +73,24 @@ const char *bgp_dest_get_prefix_str(struct bgp_dest *dest)
 }
 
 /*
- * bgp_node_create
+ * bgp_dest_unlock_node
  */
-static struct route_node *bgp_node_create(route_table_delegate_t *delegate,
-					  struct route_table *table)
+inline void bgp_dest_unlock_node(struct bgp_dest *dest)
```

Comment:
We miss `frrtrace(1, frr_bgp, bgp_dest_unlock, dest);` for this function as it was before.

Reply:
Sure, it's my mistake. I have added it now. Thanks.